### PR TITLE
Add clear storage over limit message

### PR DESCRIPTION
**Pull request by PettitWesley**
Supports: https://github.com/aws-samples/amazon-ecs-firelens-examples/pull/110

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+
+* Enhancement - Add clear info message when chunks are removed because `storage.total_limit_size` is reached [fluent-bit:6719](https://github.com/fluent/fluent-bit/pull/6719)
```
Comment (PettitWesley) [inline CHANGELOG.md:4]: Supports this key enhancement to our oomkill guide: https://github.com/aws-samples/amazon-ecs-firelens-examples/pull/110

File: AWS_FLB_CHERRY_PICKS [hunk 2]
```diff
@@ -48,3 +48,7 @@ https://github.com/PettitWesley/fluent-bit.git tail-modify-debug-context-immutab
 # S3 tag corruption fix
 https://github.com/PettitWesley/fluent-bit.git s3-str-fixes 38303131e049265277881c0d79935ad31fdd3e13
 
+# User friend message when storage.total_limit_size causes fs chunk deletion, before there was no indication
```
Comment (PettitWesley) [inline AWS_FLB_CHERRY_PICKS:51]: these were already merged in master, I checked and ran build myself, it works

Comment (matthewfala) [reply AWS_FLB_CHERRY_PICKS:51]: Cool! Thanks, this is helpful

File: CHANGELOG.md (hunk 1 above)
Comment (matthewfala) [reply CHANGELOG.md:4]: Great! This sounds good.

**Review by matthewfala — APPROVED** [review]

**Review by ShelbyZ — APPROVED** [review]

**Pull request merged**